diff --git a/dist/client.cjs b/dist/client.cjs
index a1f8b8dae234251eed1c8c89ff03dc6dd6ad2480..e47c532246be836a3422d579cb4f73ba4590d4c0 100644
--- a/dist/client.cjs
+++ b/dist/client.cjs
@@ -368,6 +368,7 @@ ${getStack()}`
       __vite_ssr_dynamic_import__: request,
       __vite_ssr_exports__: exports,
       __vite_ssr_exportAll__: (obj) => exportAll(exports, obj),
+      __vite_ssr_exportName__: (name, getter) => Object.defineProperty(exports, name, { enumerable: true, configurable: true, get: getter }),
       __vite_ssr_import_meta__: meta,
       // cjs compact
       require: node_module.createRequire(href),
diff --git a/dist/client.mjs b/dist/client.mjs
index bf8f6bc2d5afd373defe44814762d4a3bcef1cc2..8dbe2e28acc21082f7a04b68d1429d68f314beb4 100644
--- a/dist/client.mjs
+++ b/dist/client.mjs
@@ -366,6 +366,7 @@ ${getStack()}`
       __vite_ssr_dynamic_import__: request,
       __vite_ssr_exports__: exports,
       __vite_ssr_exportAll__: (obj) => exportAll(exports, obj),
+      __vite_ssr_exportName__: (name, getter) => Object.defineProperty(exports, name, { enumerable: true, configurable: true, get: getter }),
       __vite_ssr_import_meta__: meta,
       // cjs compact
       require: createRequire(href),
